locales-src/extra_aliases.js

```diff
@@ -49,6 +49,14 @@ module.exports = {
     deireadh: "end",
   },
 
+  de: {
```

Comment:
It looks like we've copied the `de` translation, and lost the `fr` translation. Would you mind putting those back? :pray: